mining/scheduler_test.go

```diff
@@ -34,17 +41,83 @@ func TestMineOnce(t *testing.T) {
 	assert.True(t, ts.ToSlice()[0].StateRoot.Equals(result.NewBlock.StateRoot))
 }
 
+// Fake worker porcelain
+type fakeWorkerPorcelain struct{}
+
+func (fwp *fakeWorkerPorcelain) MinerGetWorkerAddress(ctx context.Context, minerAddr address.Address, baseKey types.TipSetKey) (address.Address, error) {
+	return minerAddr, nil
+}
+
+func (fwp *fakeWorkerPorcelain) BlockTime() time.Duration {
+	return time.Millisecond
+}
+
+// TestMineOnce10Null calls mine once off of a base tipset with a ticket that
+// will only win 10 rounds and verifies that the output has 10 tickets and a
+// +10 height.
+func TestMineOnce10Null(t *testing.T) {
+	mockSigner, kis := types.NewMockSignersAndKeyInfo(5)
+	ki := &(kis[0])
+	addr, err := ki.Address()
+	require.NoError(t, err)
+	baseTicket := consensus.SeedFirstWinnerInNRounds(t, 10, ki, 1, 10)
```

Comment:
Use larger numbers for the miner/total power so these parameters are less opaque to read.